x/axelarnet/rate_limit.go

```diff
@@ -75,17 +77,37 @@ func NewRateLimitedICS4Wrapper(channel porttypes.ICS4Wrapper, rateLimiter RateLi
 }
 
 // SendPacket implements the ICS4 Wrapper interface
-func (r RateLimitedICS4Wrapper) SendPacket(ctx sdk.Context, chanCap *capabilitytypes.Capability, packet ibcexported.PacketI) error {
-	if err := r.channel.SendPacket(ctx, chanCap, packet); err != nil {
-		return err
+// func (r RateLimitedICS4Wrapper) SendPacket(ctx sdk.Context, chanCap *capabilitytypes.Capability, packet ibcexported.PacketI) (uint64, error) {
+func (r RateLimitedICS4Wrapper) SendPacket(
```

Comment:
why did you move the creation of the packet into the ratelimiter?

Reply:
downstreamed by `ICS4Wrapper` interface change